packages/utils/src/s3/S3Client.js

```diff
@@ -105,18 +105,29 @@ export class S3Client {
    * @param {*} [allowOverwrite]
    */
   async uploadImage(base64EncodedImage, fileId, allowOverwrite = false) {
-    const buffer = Buffer.from(
-      base64EncodedImage.replace(/^data:image\/\w+;base64,/, ''),
-      'base64',
-    );
+    const imageTypes = ['png', 'jpeg', 'jpg', 'gif', 'svg+xml'];
```

Comment:
We're slowly moving things from utils to tsutils. New stuff goes into tsutils but also big changes to existing files I think we should move over. Totally ok if you don't want to or don't have time for this one, just thought I'd mention in case you want to :)